upstream-community-operators/ffdl/ffdl-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,166 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: ffdl-operator
+      deployments:
+      - name: ffdl-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: ffdl-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: ffdl-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please set `WATCH_NAMESPACE`'s `fieldPath` value to `metadata.annotations['olm.targetNamespaces']`

Reply:
Done